vite.config.js

```diff
@@ -0,0 +1,58 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue2'
+import EnvironmentPlugin from 'vite-plugin-environment'
+import ViteRequireContext from '@originjs/vite-plugin-require-context'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
+
+const fs = require('fs')
+const packageJson = fs.readFileSync('./package.json')
+const appName = JSON.parse(packageJson).appName
+const appVersion = JSON.parse(packageJson).version
+const sbcName = JSON.parse(packageJson).sbcName
+const sbcVersion = JSON.parse(packageJson).dependencies['sbc-common-components']
+const aboutText1 = (appName && appVersion) ? `${appName} v${appVersion}` : ''
+const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
+
+// https://vitejs.dev/config/
+export default defineConfig(() => {
+  return {
+    define: {
+      'import.meta.env.ABOUT_TEXT':
+        (aboutText1 && aboutText2) ? `"${aboutText1}<br>${aboutText2}"`
+          : aboutText1 ? `"${aboutText1}"`
+            : aboutText2 ? `"${aboutText2}"`
+              : ''
+    },
+    envPrefix: 'VUE_APP_', // Need to remove this after fixing vaults. Use import.meta.env with VUE_APP.
+    plugins: [
+      vue(),
+      EnvironmentPlugin({
+        BUILD: 'web' // Fix for Vuelidate, allows process.env with Vite.
+      }),
+      ViteRequireContext() // Support require.context in vite.
+    ],
+    resolve: {
+      alias: {
+        '@': path.resolve(__dirname, './src')
+      },
+      extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json', '.vue']
+    },
+    test: {
+      // simulate DOM with happy-dom
+      environment: 'happy-dom',
```

Comment:
![image](https://github.com/bcgov/business-create-ui/assets/3484109/9ab211dd-d9b9-428e-b3cb-510547020aaa)

Reply:
cool I was using jsdom

Reply:
I've tried jsdom and it looked to me they work similarly. I guess either should be fine!